### PR TITLE
Implemented of a web app to bring up the tablet webView

### PR DESCRIPTION
**Pull request by druiz17**
When I web apps webview receives an onNewWindowRequest the appropriate webview will be created.

### CODE REVIEW

**Comment by druiz17** [comment]
Test plan 
- Test in both HMD and Desktop mode
- after taking a snapshot and having the images selected
- click on the facebook icon or twitter icon and their respective pages should appear ( for desktop mode the new page should appear as a new window, for the tablet, the new page should appear on the tablet)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2bf7b12c7dc13682424d00ad6cf7dbb9d92ee4be available at: https://gist.github.com/459e4c861195e8cc316eadc5b55555a9<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-2bf7b12.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-2bf7b12.dmg

File: interface/resources/qml/hifi/tablet/WindowRoot.qml [hunk 1]
```diff
@@ -38,6 +38,11 @@ Windows.ScrollingWindow {
         loader.source = url;
     }
 
+    function loadWebBase() {
+	loader.source = "";
```
Comment (sethalves) [inline interface/resources/qml/hifi/tablet/WindowRoot.qml]: funny indentation here

File: interface/resources/qml/hifi/tablet/TabletRoot.qml [hunk 2]
```diff
@@ -87,6 +90,11 @@ Item {
             loader.item.gotoPreviousApp = true;
         }
     }
+
+    function loadWebBase() {
+	loader.source = "";
```
Comment (sethalves) [inline interface/resources/qml/hifi/tablet/TabletRoot.qml]: indentation

**Comment by hifi-gustavo** [comment]
Build console output for commit: 763feccbe104fab542d80fcf67dc1d38188d8ae0 available at: https://gist.github.com/459e4c861195e8cc316eadc5b55555a9<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-763fecc.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-763fecc.dmg

**Comment by druiz17** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 763feccbe104fab542d80fcf67dc1d38188d8ae0 available at: https://gist.github.com/459e4c861195e8cc316eadc5b55555a9<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-763fecc.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-763fecc.dmg

**Comment by claudioshg** [comment]
testing

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-763fecc.exe

FAIL:
- Testing in HMD and Desktop Mode was a FAIL. After taking a SNAP, the option to select the Facebook and Twitter icon's did not display.
-------------
HMD MODE
xcpr10335
DESKTOP MODE
xbpr10335

**Comment by druiz17** [comment]
@claudioshg the images are not selected by default. you have to go to the home screen and back to snapshot and you should be able to select them.

**Comment by zfox23** [comment]
@druiz17 - that's not quite right.

@claudioshg how long are you waiting between seeing the "Snapshots" app come back up and deeming this a FAIL?

In the current UX of the Snap app (which isn't part of this PR and should not be considered final), the user has to wait a bit for the snapshot to upload to the High Fidelity servers before they can SHARE them. That takes a non-zero amount of time (between 5 seconds to 30 seconds, probably).

**Comment by zfox23** [comment]
@druiz17 I just tried QAing this myself - one thing I noticed is that the tap targets beneath the popup WebView are still active. An easy way to confirm this is to bring up the Twitter/FB page from the Snapshot Review dialog, then try clicking in the top right of the WebView. You'll notice that the Settings screen appears.

**Comment by claudioshg** [comment]
I retested this PR. I got around the issue following @druiz17 advice. After taking a SNAP, go back to HOME, then go back to SNAP, and the images were able to be selected. The Facebook and Twitter icons displayed. Clicking on the icons took me to their respective pages. This worked for both Desktop and HMD modes.
This PR is a PASS.

**Comment by hifi-gustavo** [comment]
Build console output for commit: bb9eb986257177f0025975a10e09eee15a30e04b available at: https://gist.github.com/459e4c861195e8cc316eadc5b55555a9<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-bb9eb98.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-bb9eb98.dmg

**Comment by druiz17** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: bb9eb986257177f0025975a10e09eee15a30e04b available at: https://gist.github.com/459e4c861195e8cc316eadc5b55555a9<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-bb9eb98.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10335-bb9eb98.dmg

**Pull request merged**